src/src/Mastermind.java

```diff
@@ -1,16 +1,42 @@
+import java.util.ArrayList;
 
 class Mastermind {
+	private ArrayList<Attempt> attempts = new ArrayList<Attempt>();
+	private SecretCombination secretCombination;
+	private static final int NUM_ATTEMPS = 10;
+	
 
 	public Mastermind() {
-		
+		secretCombination = new SecretCombination();
 	}
 	
 	public void play() {
+		Console console = new Console();
+		int nAttemps = 0;
+		console.write(nAttemps + " attemp(s):");
+		console.write("xxxx");
+		do {
+			attempts.add(new Attempt(secretCombination));
+			console.write(nAttemps + 1  + " attemp(s):");
+			console.write("xxxx");
```

Comment:
Sustituir por el método
```suggestion
			console.write(this.secretCombination.toString());
```